Makefile

```diff
@@ -8,7 +8,7 @@ endif
 COMMON_GOFLAGS := -mod=vendor
 COMMON_LDFLAGS := -X $(PROJECT)/pkg/version.GITCOMMIT=$(GITCOMMIT)
 BUILD_FLAGS := $(COMMON_GOFLAGS) -ldflags="$(COMMON_LDFLAGS)"
-CROSS_BUILD_FLAGS := $(COMMON_GOFLAGS) -ldflags="-s -w -X $(PROJECT)/pkg/segment.writeKey=R1Z79HadJIrphLoeONZy5uqOjusljSwN $(COMMON_LDFLAGS)"
+RELEASE_BUILD_FLAGS := $(COMMON_GOFLAGS) -ldflags="-s -w -X $(PROJECT)/pkg/segment.writeKey=R1Z79HadJIrphLoeONZy5uqOjusljSwN $(COMMON_LDFLAGS)"
```

Comment:
This is a different issue from the one addressed here, but I was wondering if it is safe to have this key public.

I didn't see anything on Segment website, but saw [this article](https://help.klaviyo.com/hc/en-us/articles/115005252988-How-to-Integrate-with-Segment) saying:

> Your write key is another private API key. Treat it like a password; keep it in a safe place and never expose it to the public.

Reply:
If we want people to build the production `odo` binary from our public source code there is no way to make this not public.

I don't think that this key should be treated as a secret, the most that users can do with this is to send fake data to our telemetry, but no-one will be able to access any data with this key.